x-pack/test/ingest_manager_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -200,6 +203,7 @@ export default function (providerContext: FtrProviderContext) {
     describe('uninstalls all assets when uninstalling a package', async () => {
       skipIfNoDockerRegistry(providerContext);
       before(async () => {
+        await installPackage(pkgKey);
```

Comment:
This looks like it adds then immediately removes the package. Is that what's happening?

Reply:
yeah this is just ensuring that the package was installed the first time. I got in a weird state locally when running the tests where the package was never actually installed the first time.